cupyx/scipy/signal/signaltools.py

```diff
@@ -0,0 +1,72 @@
+import math
+
+
+def choose_conv_method(in1, in2, mode='full'):
+    """Find the fastest convolution/correlation method.
+
+    Args:
+        in1 (cupy.ndarray): first input.
+        in2 (cupy.ndarray): second input.
+        mode (str, optional): ``valid``, ``same``, ``full``.
+
+    Returns:
+        str: A string indicating which convolution method is fastest,
+        either ``direct`` or ``fft1``.
+
+    .. warning::
+        This function currently doesn't support measure option,
+        nor multidimensional inputs. It differs from SciPy's
+        behavior in handling signed and unsigned integers by
+        always choosing ``direct`` convolution method.
```

Comment:
How about writing just `This function does not guarantee the compatibility of the return value to NumPy's one.`?